cli/command/formatter/history.go

```diff
@@ -78,7 +78,7 @@ func (c *historyContext) ID() string {
 }
 
 func (c *historyContext) CreatedAt() string {
-	return time.Unix(c.h.Created, 0).Format(time.RFC3339)
+	return time.Unix(c.h.Created, 0).UTC().Format(time.RFC3339)
```

Comment:
I'm actually not quite sure on this one. But I think we can make the assumption it's in `UTC` timezone.

Reply:
I don't know, it seems like this will change the value users are seeing from local time to UTC, which will be confusing if all the other times they see are local.

Reply:
hm, yes, good point